required/latex-lab/testfiles/everymath-001.tlg

```diff
@@ -0,0 +1,6 @@
+This is a generated file for the l3build validation system.
+Don't change this file in any respect.
+ y = mx + c
+y = mx + c
```

Comment:
what's the extra space doing here?

Reply:
`\begin{equation}` is followed by a newline in my test file, so that's a space ;) The aim is to capture the tokens exactly as TeX would see them: we likely want to do further processing, but I've not included that at present.

Reply:
isn't the amsmath grabbing cleaning those newlines out? if so we may want to do that for consistency too ... but it isn't really that important, just noticed the difference in the output.